hedera-node/test-clients/src/main/resource/contract/contracts/Module049OpcodesExist/Module049OpcodesExist.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.24;
+
+contract Module049OpcodesExist {
+    function try_transient_storage() public {
+        assembly {
+            tstore(10, 11111)
+            pop(tload(10))
+        }
+    }
+
+    function try_mcopy() public pure {
+        bytes memory byts = "Hello, World!";
+        assembly {
+           mcopy(0,0,32)
```

Comment:
Q: is there a way to confirm the copy location?

Reply:
(Discussed offline, result:). These opcodes (MCOPY, TLOAD, TSTORE) are implemented by BESU itself and adopted here _without change_.  BESU has extensive tests itself (and also runs standard Ethereum EVM tests).  I consider it sufficient, here, to simply test that the opcodes exist (execute without causing an invalid operation exception) in this (Cancun) module and _don't_ exist in the previous module.